ydb/library/yql/dq/actors/task_runner/task_runner_actor_local.cpp

```diff
@@ -165,6 +167,7 @@ class TLocalTaskRunnerActor
                 TaskRunner->SetWatermarkIn(watermark);
             }
 
+            TaskRunner->SetSpillerFactory(std::make_shared<TDqSpillerFactory>(TxId, NActors::TActivationContext::ActorSystem(), WakeUpCallback));
```

Comment:
Why do we need to call this Setter multiple times?

Reply:
You're right, thanks
It should be moved